leofs-adm

```diff
@@ -67,6 +67,14 @@ case $OS in
     SunOS)
         NC_ARGS=""
         ;;
+    Linux)
+        ## For Ubuntu-18.04
+        if cat /etc/issue | grep -q 'Ubuntu Bionic'; then
```

Comment:
`/etc/issue` can be modified by sysadmin so it would be better to use `lsb_release` to distinguish the distribution and its version like the following.

```shell
leofs@cat2neat$ lsb_release -is
Ubuntu
leofs@cat2neat$ lsb_release -rs
18.04
leofs@cat2neat:leo_object_storage$ lsb_release -irs
Ubuntu
18.04
```